com.unity.render-pipelines.high-definition/Runtime/Lighting/ScreenSpaceLighting/HDRenderPipeline.ScreenSpaceGlobalIllumination.cs

```diff
@@ -196,9 +196,9 @@ TraceOutput TraceSSGI(RenderGraph renderGraph, HDCamera hdCamera, GlobalIllumina
 
                 // History buffers
                 var colorPyramid = hdCamera.GetPreviousFrameRT((int)HDCameraFrameHistoryType.ColorBufferMipChain);
-                passData.colorPyramid = colorPyramid != null ? builder.ReadTexture(renderGraph.ImportTexture(colorPyramid)) : renderGraph.defaultResources.blackTextureXR;
+                passData.colorPyramid = hdCamera.colorPyramidHistoryIsValid && colorPyramid != null ? builder.ReadTexture(renderGraph.ImportTexture(colorPyramid)) : renderGraph.defaultResources.blackTextureXR;
                 var historyDepth = hdCamera.GetCurrentFrameRT((int)HDCameraFrameHistoryType.Depth);
-                passData.historyDepth = historyDepth != null ? builder.ReadTexture(renderGraph.ImportTexture(historyDepth)) : renderGraph.defaultResources.blackTextureXR;
+                passData.historyDepth = hdCamera.colorPyramidHistoryIsValid && historyDepth != null ? builder.ReadTexture(renderGraph.ImportTexture(historyDepth)) : renderGraph.defaultResources.blackTextureXR;
```

Comment:
Not sure about the condition here

Reply:
@anisunity yep the test failed again. Its unfortunate because I cant reproduce this issue locally. Only happens on the CIs (could be because I have a neo kit wiht ps4 emulation?)
Anyway, changing this to your suggestion and launching the CI again to see if it worked.